openedx/core/djangoapps/programs/utils.py

```diff
@@ -158,7 +159,7 @@ def progress(self, programs=None, count_only=True):
                 'uuid': program['uuid'],
                 'completed': len(completed) if count_only else completed,
                 'in_progress': len(in_progress) if count_only else in_progress,
-                'not_started': len(not_started) if count_only else not_started,
```

Comment:
Nitpick: please leave this trailing comma, it's good practice. It leaves the next person to touch this dictionary with a cleaner diff.